docs/Open-source/contributing.md

```diff
@@ -10,6 +10,7 @@ tags:
 - repository
 - contribute
 - open source
+hide_table_of_contents: true
```

Comment:
more out of curiosity, why do we want to hide the table of contents? 🤔

Reply:
just double checked and I'm bringing it back :)

Reply:
thanks for checking! 🙏